src/utils/dates.js

```diff
@@ -1,18 +1,34 @@
-const moment = require("moment-timezone");
+const { Temporal } = require("@js-temporal/polyfill");
 const holidays = require("@18f/us-federal-holidays");
 
-const getNextHoliday = (timezone = "America/New_York") => {
-  const now = moment.tz(timezone);
+const getNextHoliday = (timeZone = "America/New_York") => {
+  const now = Temporal.Now.instant().toZonedDateTime({
+    calendar: "iso8601",
+    timeZone,
+  });
 
-  return holidays
-    .allForYear(now.year())
-    .concat(holidays.allForYear(now.year() + 1))
-    .map((h) => ({
-      ...h,
-      date: moment.tz(h.dateString, "YYYY-MM-DD", timezone),
-    }))
-    .filter((h) => h.date.isAfter(now))
-    .shift();
+  return (
+    holidays
+      .allForYear(now.year)
+      .concat(holidays.allForYear(now.year + 1))
+      .map((h) => {
+        const [year, m, d] = h.dateString.split("-");
+        const month = m < 10 ? `0${m}` : m;
+        const day = d < 10 ? `0${d}` : d;
```

Comment:
starting in node 8, looks like [String.padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) is available?

```suggestion
        const month = String(m).padStart(2, '0');
        const day = String(d).padStart(2, '0');
```

Reply:
YES! Thank you! I *always* forget that `padStart` is part of the spec now.